components/onboarding/Welcome.js

```diff
@@ -2,18 +2,21 @@ import { Heading, Copy, Button } from "./styles"
 
 import progress1 from "../../static/images/progress1.svg"
 
-// TODO: tell people not to press back and forward buttons
-
 const Welcome = ({ incrementPage, employer }) => (
   <main className="flex justify-center items-center flex-col w-5/12 m-auto">
     <Heading className="mb-6 self-start">Welcome to Catapillr!</Heading>
-    <Copy className="mb-12">
+    <Copy className="mb-6">
       We've partnered up with <span className="font-bold">{employer.name}</span>{" "}
       to offer you interest-free childcare loans to help tide you over and
       hopefully make both your work life and your home life more rewarding.
       Press the button below to see how much you could borrow from your
       employer!
     </Copy>
+    <Copy className="mb-12">
+      During the application process, please don't use the buttons on your
```

Comment:
I do think we should handle this differently, so people [are able to use their back/forward buttons](https://github.com/Catapillr/loan-application-platform/issues/82)